pycares/_cfficore/__init__.py

```diff
@@ -651,7 +662,7 @@ def __init__(self, srv):
 
 class ares_query_txt_result(object):
     def __init__(self, txt):
-        self.txt = _ffi_string(txt.txt)
+        self.text = _ffi_string(txt.txt)
```

Comment:
I'm guessing this will break code for people using CFFI? As they would've been using "txt" instead of "text" property on their results in the previous versions. Is that a problem?